src/pybind/mgr/dashboard/frontend/src/app/core/auth/auth.module.ts

```diff
@@ -28,7 +28,11 @@ import { UserTabsComponent } from './user-tabs/user-tabs.component';
     NgbPopoverModule,
     NgxPipeFunctionModule,
     RouterModule,
-    NgbModule
+    NgbModule,
+    IconModule,
+    GridModule,
+    ButtonModule,
+    InputModule
```

Comment:
Can you check if we can export these modules from shared module if they are already available!

Reply:
good point, even though we have some of those imports added to the SharedModule or rather the ComponentsModule none of them are exported which means that it won't be reusable anywhere else. So if we want to use them we'll need to properly export them. however considering we don't yet do that in the dashboard anywhere, maybe we can follow the usual pattern here as well?
TBH, if we can we should deprecate the NgModules thing and move to [standalone components](https://v17.angular.io/guide/standalone-components), which can keep the whole import mess under control, but that's a different "future" discussion.

Reply:
we are not exporting anything from shared module in existing code following same